core/controllers/editor.py

```diff
@@ -744,7 +756,8 @@ def put(self, exploration_id):
         self.render_json({
```

Comment:
Done.

Reply:
Can you add information about the expected behavior and what changes you have made? [Asking this as this thread is different and needs clear information, this will help other reviewers as well.]

Reply:
After this PR, the draft will not be saved whenever the changes coming from an old version from the frontend are not mergeable to the latest version at the backend. So initially comment was an old one where we used to save the draft in case of version invalid. Now I changed the comment to explain the current situation.